templates/bastion-and-eks-cluster.yaml

```diff
@@ -753,9 +751,10 @@ Resources:
       AutoScalingRollingUpdate:
         MinInstancesInService: !Ref MinNumberOfNodes
         MaxBatchSize: !Ref MaxNumberOfNodes
-    # CreationPolicy:
-    #   ResourceSignal:
-    #     Timeout: PT40M
+    CreationPolicy:
+      ResourceSignal:
+        Count: !Ref DesiredNumberOfNodes
+        Timeout: PT15M
```

Comment:
`Timeout: PT15M` seems OK, but may be `25M` can give some extra time.

Reply:
I made it 30M so it's consistent with the bastion configuration.

Reply:
Great.  Thanks.